.gitignore

```diff
@@ -21,6 +21,9 @@ bld/
 [Oo]bj/
 msbuild.log
 
+# Visual Studio 2015
+.vs/
+
 # Roslyn stuff
```

Comment:
:bulb: The definition above *replaces* the items currently listed under `Roslyn stuff`. Visual Studio "14" CTPs used to create a folder **{SolutionName}.sln.ide**, but starting with CTP 6 it creates a folder just called **.vs**.